Tests/YamsTests/PerformanceTests.swift

```diff
@@ -57,90 +57,137 @@ class PerformanceTests: XCTestCase {
         "/SourceKitten/Source/SourceKittenFramework/Xcode.swift"
     ]
 
-    func testSourceKittenIssue289UsingLoad() {
-        guard let yamlString = try? String(contentsOfFile: filename, encoding: .utf8) else {
+    func loadYAML() throws -> String {
+        let data = try Data(contentsOf: URL(fileURLWithPath: filename))
```

Comment:
Why not use `String(contentsOfFile:encoding:)` like before?

```swift
func loadYAML() throws -> String {
    return try String(contentsOfFile: filename, encoding: .utf8)
}
```

Reply:
As far as I tested, `String.UTF8View.withContiguousStorageIfAvailable(_:)` does not work if `String` is created by `String.init(contentsOfFile:)` on Apple Platform.

Reply:
That seems odd but okay.